packages/contracts/src/RelayHub.sol

```diff
@@ -382,6 +407,10 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
         bool rejectOnRecipientRevert;
     }
 
+    /**
+     * @notice This method can only by called by this `RelayHub`.
```

Comment:
its a "@dev" (that is, this method should be "developer-only" function, or internal. but it has to be "external", so at least not document it as a real external function)